frontend/src/views/WheresMyGene/components/GeneSearchBar/components/QuickSelect/index.tsx

```diff
@@ -299,15 +307,17 @@ export default function QuickSelect<
   );
 
   function renderOption(
+    optionProps: HTMLAttributes<HTMLLIElement>,
     option: T,
     { selected }: AutocompleteRenderOptionState
   ) {
     return (
       <StyledMenuItem
-        {...{ component: "div" }}
+        {...{ component: "li" }}
```

Comment:
JW, why the change from `div` to `li` here?

Reply:
Not sure if this is related to the change from `div` to `li`, but there is this default highlight that remains on the  genes/tissues when the mouse is not hovering over an item. Also, when the selector opens up, the first item is highlighted. Is this intended or is it an unavoidable artifact? 

See video for reference:
https://user-images.githubusercontent.com/16548075/207379859-017af808-0a7e-42b2-b657-45f750fdda67.mov

Reply:
Thanks so much for catching this! I pushed a fix for the styling 🙏

And we had to use li because using div under a list (ol or ul) is technically illegal in HTML as the linter points out to me!